doc/u-boot-env.txt

```diff
@@ -0,0 +1,127 @@
+
+#### NOTE 1
+To select source of loaders, use on-board switches: SW10 on Salvator-X/XS and SW6 on StarterKit.
+
+For Salvator boards set `SW10[5..8]` to `[OFF OFF ON OFF]` for eMMC and inverted values `[ON ON OFF ON]` for HyperFlash.
+
+For StarterKit set `SW6[4]` to `[OFF]` for bootloaders on eMMC, and `[ON]` for HyperFlash.
+
+To make sure that loaders are loaded from expected place, look for the line `NOTICE:  BL2: Boot device is ` during start of the board.
+
+
+#### NOTE 2
+To activate the possibility of booting SoC from the eMMC, the content of the next EXT_CSD registers should be changed.
+This procedure should be done only once.
+```
+[179:179]  PARTITION_CONFIG                           0x08
+[177:177]  BOOT_BUS_CONDITIONS                        0x0A
+```
+Boot `Flash Writer` according to instruction https://elinux.org/R-Car/Boards/H3SK#Tips.
+
+Use EM_SECSD command to change 177(0xB1) and 179(0xB3) registers. E.g.:
+```
+>EM_SECSD
+  Please Input EXT_CSD Index(H'00 - H'1FF) :B3
+  EXT_CSD[B3] = 0x00
+  Please Input Value(H'00 - H'FF) :8
+  EXT_CSD[B3] = 0x08
+```
+
+Use following commands to flash loaders to eMMC:
+```
+setenv flash_bootparam_sa0 'tftp 0x48000000 bootparam_sa0.bin; mmc dev 1 1; mmc write 0x48000000 0x0 0x1E;'
+setenv flash_bl2 'tftp 0x48000000 bl2.bin; mmc dev 1 1; mmc write 0x48000000 0x1E 0x162;'
+setenv flash_cert_header_sa6 'tftp 0x48000000 cert_header_sa6_emmc.bin; mmc dev 1 1; mmc write 0x48000000 0x180 0x80;'
+setenv flash_bl31 'tftp 0x48000000 bl31.bin; mmc dev 1 1; mmc write 0x48000000 0x200 0xE00;'
+setenv flash_tee 'tftp 0x48000000 tee.bin; mmc dev 1 1; mmc write 0x48000000 0x1000 0x600;'
+setenv flash_u_boot 'tftp 0x48000000 u-boot.bin; mmc dev 1 2; mmc write 0x48000000 0x0 0x800;'
+setenv flash_z_loaders 'run flash_bootparam_sa0; run flash_bl2; run flash_cert_header_sa6; run flash_bl31; run flash_tee; run flash_u_boot;'
+```
+
+To flash loaders to HyperFlash (available for Kingfisher only):
+```
+setenv flash_hf_bootparam_sa0 'tftp 0x48000000 bootparam_sa0.bin; erase 0x08000000 +0x${filesize}; cp.b 0x48000000 0x08000000 0x${filesize};'
+setenv flash_hf_bl2 'tftp 0x48000000 bl2.bin; erase 0x08040000 +0x${filesize}; cp.b 0x48000000 0x08040000 0x${filesize};'
+setenv flash_hf_cert_header_sa6 'tftp 0x48000000 cert_header_sa6.bin; erase 0x08180000 +0x${filesize}; cp.b 0x48000000 0x08180000 0x${filesize};'
+setenv flash_hf_bl31 'tftp 0x48000000 bl31.bin; erase 0x081C0000 +0x${filesize}; cp.b 0x48000000 0x081C0000 0x${filesize};'
+setenv flash_hf_tee 'tftp 0x48000000 tee.bin; erase 0x08200000 +0x${filesize}; cp.b 0x48000000 0x08200000 0x${filesize};'
+setenv flash_hf_u_boot 'tftp 0x48000000 u-boot.bin; erase 0x08640000 +0x${filesize}; cp.b 0x48000000 0x08640000 0x${filesize};'
+setenv flash_hf_cr7 'ftp 0x48000000 cluster_wrapper.bin; erase 0x0A000000 +0x${filesize}; cp.b 0x48;'
```

Comment:
Please fix this line
- 't' is missed in tftp
- last command is incorrect, symbolsare missing

This line should be:

setenv flash_hf_cr7 'tftp 0x48000000 cluster_wrapper.bin; erase 0x0A000000 +0x${filesize}; cp.b 0x48000000 0x0A000000 0x${filesize};'